dvc/remote/local/__init__.py

```diff
@@ -313,13 +315,13 @@ def status(
         show_checksums=False,
         download=False,
     ):
-        logger.info(
+        logger.debug(
```

Comment:
Why are these downgraded to `.debug()`? This will not show them by default.

Reply:
yes; seems to be best @efiop?

Reply:
@casperdcl Looks good to me. These messages were necessary a long time ago when no_traverse config option was set to false and we didn't have progress bars notifying users about what is going on. With new progress bars, we don't really need these messages in info output.

Reply:
@casperdcl Btw, maybe you could use asciinema to show us before and after your pr? It would be really informative.

Reply:
[![asciicast](https://asciinema.org/a/jDmhBlsa7TPY88GRE56WHqJYk.svg)](https://asciinema.org/a/jDmhBlsa7TPY88GRE56WHqJYk)